contrib/packaging/deb/cfg/debian/control

```diff
@@ -0,0 +1,19 @@
+Source: cilium
+Maintainer: Andre Martins <andre@cilium.io>
+Section: net
+Priority: optional
+Standards-Version: ${VERSION}
+Homepage: https://github/cilium/cilium
+Vcs-Git: git://github/cilium/cilium.git
+
+Package: cilium
+Architecture: amd64
+Section: net
+Priority: optional
+Depends: docker-engine (>=1.12), clang-3.8, libc6-dev-i386, iproute2 (>=4.6.0),
+ linux-image-4.8.1-040801-generic
```

Comment:
No, the version is in the name of the kernel package. We can omit it and write in **bold** the minimal kernel version that cilium needs